src/web-auth/captcha.js

```diff
@@ -0,0 +1,117 @@
+// eslint-disable-next-line no-unused-vars
+import Authentication from '../authentication';
+import object from '../helper/object';
+
+var defaults = {
+  lang: 'en',
+  templates: {
```

Comment:
@stevehobbsdev shouldn't the customization of these be documented in this repo somewhere? especially those English messages a few lines below.

Reply:
I plan to document this in the auth0 docs site, in the bot detection section